app/services/DateService.scala

```diff
@@ -130,7 +130,7 @@ class DateService @Inject()(
             PreviousScheme.OSSU,
             None,
             countryCode
-          )
+          )(hc, request.toAuthenticatedOptionalDataRequest)
```

Comment:
why do we need to do `toAuthenticatedOptionalDataRequest` ? Wouldn't it have been easier to make a alias of searchScheme, one that accepts the optional and one that accepts the non-optional? And deal with the option uniquely, looking at what originally uses the request header - the audit call- I don't even see that using `request.userAnswers` anyway, I feel adding this (and other) optional conversation is therefore redundant